kotlinx-coroutines-core/common/src/flow/operators/Limit.kt

```diff
@@ -23,7 +23,7 @@ public fun <T> Flow<T>.drop(count: Int): Flow<T> {
     return flow {
         var skipped = 0
         collect { value ->
-            if (++skipped > count) emit(value)
+            if (skipped >= count) emit(value) else ++skipped
```

Comment:
`skipped` cannot become greater than `count`

Reply:
That's a good engineering practice to write `>=` in this case anyway, even though writing `==` would technically suffice.